lib/gpprmsya.gi

```diff
@@ -1430,6 +1430,42 @@ local P;
   fi;
 end);
 
+
+BindGlobal( "NormalizerSimpleGroupInParentSymmetricViaAutomorphisms",
+function( T, aut... ),
```

Comment:
I think we need `T` to be transitive.

Reply:
So we assume that `T` is finite non-abelian and simple. _If_  the action is faithful, it is either transitive or trivial. As we compute the normalizer in the symmetric group on the moved points, I guess we want to assume that the action of `T` on its moved points is faithful? If so, the only thing we need to check is that the action is not trivial. What do you think @ssiccha ?

Reply:
I don't think `ConjugatorIsomorphism` requires `T` to be transitive, so this should not cause any problems.